src/dswx_sar/masking_with_ancillary.py

```diff
@@ -607,7 +607,6 @@ def extend_land_cover(landcover_path,
         excluded_rg_path,
         geotransform=metainfo['geotransform'],
         projection=metainfo['projection'],
-        description='Water classification (WTR)',
```

Comment:
Are the descriptions in these products not needed because they're temporary products written to scratch?